src/harness/certs/README.md

```diff
@@ -24,6 +24,13 @@ https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/
                    sas_expired                                               domain_proxy_expired
                                                                              domain_proxy_inapplicable
 
+                  -------------root_ca --------------------------------------                
```

Comment:
Maybe use `root_ca (same as above, separate graph for clarity)` or something similar, to make it clear that there's only one `root_ca`?

Reply:
Update   the graph based on the suggestion